Thanks for the test! I see that this bug was fixed similarly during a small refactoring in the migration to 10.0 indeed: https://github.com/OCA/account-financial-tools/commit/a605243de363bf8e70e1975f70f8c3e16d49be6e#diff-0f6d138cc54bb8bedfd56c2824ac233fR122

Can you follow up Travis' comment?

```
account_reversal/tests/test_account_reversal.py:75:46: E712 comparison to True should be 'if cond is True:' or 'if cond:'
```

I know it won't make tests green because there is another module in this repo which is broken.

--- a/account_reversal/tests/test_account_reversal.py
+++ b/account_reversal/tests/test_account_reversal.py
@@ -63,11 +63,16 @@ class TestAccountReversal(TransactionCase):
             ':SALE_' or ':CUSTOMER_')
             for x in move.line_ids.sorted(key=lambda r: r.account_id.id)])
 
-    def test_reverse(self):
+    def test_reverse_posted_reconcile(self):
         move = self._create_move()
         self.assertEqual(
             self._move_str(move), '0.00100.00:SALE_100.000.00:CUSTOMER_')
-        rev = move.create_reversals()
+        rev = move.create_reversals(post=True, reconcile=True)
         self.assertEqual(len(rev), 1)
         self.assertEqual(
             self._move_str(rev), '100.000.00:SALE_0.00100.00:CUSTOMER_')
+        reconciled_line = move.line_ids.filtered(
+            lambda l: l.account_id.reconcile)
+        self.assertEqual(
+            'posted', rev.state)
+        self.assertTrue(reconciled_line.full_reconcile_id)

